definitions/features/katello_service.rb

```diff
@@ -27,13 +29,65 @@ def make_start(spinner, options = {})
       spinner.update 'No katello service to start'
     else
       filters = "--only #{services.join(',')}"
-      spinner.update 'Starting the katello services..'
+      spinner.update 'Starting katello services..'
       execute!("katello-service start #{filters}")
     end
   end
 
+  def restart(spinner, options = {})
+    spinner.update 'Restarting katello services..'
```

Comment:
Given spinner is only being used here, what about moving this call to the procedure itself, so that the restart method doesn't have to know about the spinner at all?